crates/exex/exex/src/dyexex.rs

```diff
@@ -0,0 +1,32 @@
+//! Dynamically loading type-safe abstractions
+
+/// Dynamically loaded ExEx entrypoint, that accepts the [`ExExContext`](`reth_exex::ExExContext`)
+/// and returns a Future that will be polled by the [`ExExManager`](`reth_exex::ExExManager`).
```

Comment:
hmm, it doesn't actually accepts an `ExExContext`, but rather a function that accepts one. Can we clarify the doc here?

Reply:
@shekhirin can be

```
/// Dynamically loads an ExEx entrypoint, which accepts a user-defined function representing the core ExEx logic. 
/// The provided function must take an [`ExExContext`](`crate::ExExContext`) as its argument
```